elements/style.html

```diff
@@ -38,9 +38,13 @@
   </div>
   <pre class="code-example">
     <code>
-      &lt;head&gt;
-        &lt;h1&gt;<strong>&lt;style&gt;</strong>This content is styled in CSS<strong>&lt;style&gt;</strong>&lt;h1&gt;
-      &lt;/head&gt;
+      &lt;style&gt;
```

Comment:
you've lost the highlighting in the code example

![image](https://cloud.githubusercontent.com/assets/9439053/12464516/f020cd6e-bf8e-11e5-8036-6e31b5fda3cc.png)